api/handlers.go

```diff
@@ -39,6 +40,19 @@ func getInfo(c *context, w http.ResponseWriter, r *http.Request) {
 		MemTotal:          c.cluster.TotalMemory(),
 	}
 
+	if httpProxy := os.Getenv("http_proxy"); httpProxy != "" {
```

Comment:
Is the `if` really necessary? Wouldn't `HttpProxy: os.Getenv("http_proxy")` above have the same resullt?